app/Services/Forge/Pipeline/CreateDatabase.php

```diff
@@ -28,8 +29,13 @@ public function __invoke(ForgeService $service, Closure $next)
             return $next($service);
         }
 
+        if ( $service->setting->dbName ) {
+            $dbName = FormattedBranchName::run($service->setting->dbName);
+        } else {
+            $dbName = $service->getStandardizedBranchName();
+        }
+
         $dbPassword = Str::random(16);
-        $dbName = $service->setting->dbName ?? $service->getStandardizedBranchName();
```

Comment:
It turns out I created an action called "GenerateStandardizedBranchName" that generates the database name from the branch name, removing the first digit of string the db name as well. It would be great if we can get db name from the **getFormattedBranchName** through the **GetStandardizedBranchName** (perhaps with a name change in the **GetStandardizedBranchName** to **getFormattedDatabaseName**) and it will be safe :) Thanks again for your time!

Reply:
I've updated my PR to include use the GenerateStandardizedBranchName action instead of Laravel's Str helper. I've created a new `getFormattedDatabaseName` method in the ForgeService to generate the new DB name, with the formatted branch name as the fallback.